packages/nuxt/types.d.mts

```diff
@@ -27,6 +28,7 @@ declare module 'nitropack' {
     experimentalNoScripts?: boolean
   }
   interface NitroRuntimeHooks {
+    'dev:ssr-logs': (ctx: { logs: LogObject[], path: string }) => void | Promise<void>
```

Comment:
(when adding feature to nitro, i would probably use `dev:logs` so should be fine only mentioning the idea)

Reply:
would you like me to make a pr to add this to nitro?

Reply:
surely would love if you like to help. i probably make a tracker issue once enabled ALS first globally which i desperately hope to do 😆